deployer/templates/fluentd.yaml

```diff
@@ -343,4 +343,4 @@ parameters:
 -
   description: 'Fluentd buffer size limit'
   name: BUFFER_SIZE_LIMIT
-  value: "16777216"
+  value: "1048576"
```

Comment:
it's only used for testing, and only until we get rid of the need to use the logging-fluentd-template and rely directly on the daemonset config.